lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -107,10 +109,8 @@ $read_only = Get-AnsibleParam -obj $params -name "read_only" -type "bool" -defau
 $site_name = Get-AnsibleParam -obj $params -name "site_name" -type "str" -failifempty $read_only
 
 $state = Get-AnsibleParam -obj $params -name "state" -validateset ("domain_controller", "member_server") -failifempty $result
-$log_path = Get-AnsibleParam -obj $params -name "log_path"
-$_ansible_check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -default $false
 
-$global:log_path = $log_path
```

Comment:
This looks like a change in how the module behaves. This should not be part of PSLint fixes.

Reply:
Same as with win_domain_controller